src/http_server.c

```diff
@@ -1090,7 +1090,7 @@ static void http_broadcast_send(struct http_server_context *ctx, char const *msg
             mg_set_timer(nc, mg_time() + KEEP_ALIVE); // reset keep alive timer
         }
         else if (cctx && !cctx->is_chunked) {
-            mg_send(nc, msg, len);
+            mg_send(nc, msg, (int)len);
```

Comment:
This one should not be required if mongoose was consistent in its methods declarations: `mg_send_http_chunk`, `mg_send_websocket_frame` and almost all others take a `size_t`, but for some reason `mg_send` does not...